paradise_papers_django/search/static/js/search.js

```diff
@@ -9,3 +9,34 @@ function handleListVisibility (event, template) {
     container.show();
     container.toggleClass('show');
 }
+
+function changeTabs(current, other) {
+	var currentTab = $(current);
+	var otherTab = $(other);
+	if ( currentTab.hasClass("not-active"))  {
+		console.log("has class");
+		currentTab.removeClass("not-active");
+		otherTab.addClass("not-active");
+	}
+
+}
+var juristictionTab = $('.search-by-jurisdiction');
+var CountryTab = $('.search-by-country');
+
+juristictionTab.click(function(e) {
+    if ( $( this ).hasClass( "not-active" ) ) {
```

Comment:
Feedback, try to be consistent. Example: if you start adding an extra space between parenthesis add it everywhere or just not add the space. This is just conversion, your code should work anyway.